packages/web-components/src/components/cbp-structured-list/cbp-structured-list.scss

```diff
@@ -98,11 +104,18 @@ cbp-structured-list {
     padding: var(--cbp-structured-list-item-padding);
     border-bottom-style: solid;
     border-bottom-width: var(--cbp-border-size-md);
-    border-bottom-color: var(--cbp-color-gray-cool-30);
+    border-bottom-color: var(--cbp-structured-list-border-color);
 
     &:hover {
       background-color: var(--cbp-structured-list-color-background-hover) !important;
     }
   }
 
+  /* 
+    *Collection 
+  */
+  
+  div[role=list]:only-child > cbp-structured-list-item:first-of-type {
+    border-top: 1px solid var(--cbp-structured-list-border-color);
```

Comment:
You're using a magic number for the border-width instead of the token and they are different. Should be `var(--cbp-border-size-md);`